server/routes/auth/auth.route.js

```diff
@@ -34,10 +34,17 @@ router.post('/register', (req, res) => {
         .then((user) => {
           const payload = { id: user.id };
           const token = user.signJWT(payload);
+          const options = {
+            httpOnly: true,
```

Comment:
Don't you need to set the expiration here too? Here we previously had:
`expires: new Date(
              Date.now() + tokenExpiration * 24 * 60 * 60 * 1000
            ),`

Reply:
I believe that once the `jwt` token expires (expiration listed in `models/user.js`), `passport` should dis-authenticate the user and asks them to login/register. So having it just there should be fine.